<launch>
    <node pkg="sbg_driver" name="sbg_ellipseA" exec="sbg_device">
      <param from="$(find-pkg-share sbg_driver)/config/example/ellipse_A_default.yaml"/>
    </node>
</launch>